src/main/java/se/sundsvall/disturbance/integration/db/specification/DisturbanceSpecification.java

```diff
@@ -17,7 +17,7 @@ public interface DisturbanceSpecification {
 	static Specification<DisturbanceEntity> withCategoryFilter(List<Category> categoryList) {
 		return (disturbanceEntity, cq, cb) -> {
 			if (isNotEmpty(categoryList)) {
-				return disturbanceEntity.get("category").in(toStringList(categoryList));
+				return disturbanceEntity.get("category").in(categoryList);
```

Comment:
Make `"category"` a constant or use JPA metamodelgen?

Reply:
Will use constants for now